src/selenium/firefox/capabilities.cr

```diff
@@ -11,5 +11,6 @@ class Selenium::Firefox::Capabilities < Selenium::Capabilities
     end
 
     property args = [] of String
+    property binary String?
```

Comment:
oops! I got so excited at having a PR, that I totally missed that this is a syntax error :man_facepalming: